src/cli/repl.rs

```diff
@@ -61,18 +55,18 @@ fn check_non_zero(name: &str, x: usize) -> Result<()> {
     Ok(())
 }
 
-/// Pads the number of iterations to the first multiple of the reduction count
-/// that's equal or greater than the number of iterations
-///
-/// Panics if reduction count is zero
-fn pad_iterations(iterations: usize, rc: usize) -> usize {
-    let lower = rc * (iterations / rc);
-    if lower < iterations {
-        lower + rc
-    } else {
-        lower
-    }
-}
+// /// Pads the number of iterations to the first multiple of the reduction count
```

Comment:
What was wrong with this content, and the rest of the file!?

Reply:
`pad_iteration` is only used in `eval_expr_and_set_last_claim` which had to be largely commented about for the above reason having to do with the `fcomm` types erroneously being used in `lurk`. I can add a `#[allow(dead_code)]` if this is preferable to commenting it out.